src/main/java/ByeCommand.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Class to run the bye command.
+ */
+public class ByeCommand implements Command {
+
+    /**
+     * Executes the bye command, causing Duke to stop running.
+     *
+     * @param taskList Used by Duke to keep track of tasks.
+     * @param ui Responsible for printing to console after execution.
+     * @param storage Stores tasks in a text format.
+     */
+    public void execute(TaskList taskList, Ui ui, Storage storage) {
+        ui.showLine();
+        ui.goodbye();
+        ui.showLine();
+    }
+
+    /**
+     * Returns true if a bye command is called.
+     * Returns False otherwise.
+     *
+     * @return boolean indicating whether Duke is to stop running.
+     */
```

Comment:
Comment might be unnecessary here since the method name is quite self-explanatory